### ISSUE TITLE
Q&V: idea to save results for a specific user

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2725]
Now that we have things like queries and FTS and the ability to pull up data in Kiosk itself, would it make sense to think about making it possible for users to store things they are working on? This was a feature of the British Museum database that seemed useful. 

So, for instance, if I have a FTS open for "seals" and am looking at 1-3 contexts with seals, and then also have a query open about other finds in those contexts, and am working on an analysis that requires all of those things ... it would be useful to be able to save that ongoing analysis instead of having to reconstruct it every time I log back in.

**Comment by urapadmin** [comment]
ja, that sounds like a useful next feature once people actually use this for research.

**Comment by luizaogs** [comment]
Related to this, too, I think: #147
